File: backend/BookstoreTamAn/src/main/java/com/minhnghia/datn/BookstoreTamAn/model/News.java
package com.minhnghia.datn.BookstoreTamAn.model;

import com.fasterxml.jackson.databind.ser.Serializers;
import jakarta.persistence.*;
import lombok.*;

import java.util.Date;

@Entity
@Table(name = "news")
@Setter
@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class News extends BaseEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    private String title;
    private String content;
    private String imageUrl;
    private String createdBy;
    private String modifyBy;
}

File: frontend/frontend/src/app/models/news.ts
export class News{
    id : 0;
    title : string;
    content : string;
    imageUrl : string;
    createdBy : string;
    createdAt : string;
    modifyBy : string;
    modifyAt : string;

    constructor(){
        this.id = 0;
        this.title = "",
        this.content = "",
        this.imageUrl = "",
        this.createdBy = "",
        this.createdAt = "",
        this.modifyBy = "",
        this.modifyAt = ""
    }
}
export interface INewsList{
        id : 0;
    title : string;
    content : string;
    imageUrl : string;
    createdBy : string;
    createdAt : string;
    modifyBy : string;
    modifyAt : string;
}